_dev/src/router/index.ts

```diff
@@ -80,22 +74,23 @@ const routes: Array<RouteConfig> = [
     path: '/campaign',
     name: 'campaign',
     component: CampaignPage,
-    beforeEnter: campaignsAlreadyExist,
-  },
-  {
-    path: '/campaign/list',
-    name: 'campaign-list',
-    component: CampaignPage,
-  },
-  {
-    path: '/campaign-creation',
-    name: 'campaign-creation',
-    component: CampaignPage,
-  },
-  {
-    path: '/campaign-edit/:id',
-    name: 'campaign-edition',
-    component: CampaignPage,
+    children: [
+      {
+        path: 'list',
+        name: 'campaign-list',
+        component: CampaignList,
+      },
+      {
+        path: 'creation',
+        name: 'campaign-creation',
+        component: CampaignCreation,
```

Comment:
With this name we expect tu find another component `CampaignEdition`. `CampaignForm` would be less confusing?

Reply:
It's true, I'll update with a new name